golang-analyze-and-submit-to-sonar/action.yaml

```diff
@@ -23,27 +23,12 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Get temporary AWS Credentials
-      env:
-        VAULT_ADDR: ${{ inputs.vault_addr }}
-        VAULT_TOKEN: ${{ inputs.vault_token }}
-        AWS_REGION: us-west-2
-        AWS_DEFAULT_REGION: us-west-2
-      run: |
-        wget --quiet https://releases.hashicorp.com/vault/1.5.5/vault_1.5.5_linux_amd64.zip
-        unzip vault_1.5.5_linux_amd64.zip
-        chmod +x ./vault
-        export CREDS=$(./vault write -format json aws/hosted-services/sts/armory-cloud-github-actions-role ttl=60m || exit 1)
-        AWS_ACCESS_KEY_ID=$(echo $CREDS | jq -r ".data.access_key")
-        AWS_SECRET_ACCESS_KEY=$(echo $CREDS | jq -r ".data.secret_key")
-        AWS_SESSION_TOKEN=$(echo $CREDS | jq -r ".data.security_token")
-        echo "::add-mask::${AWS_ACCESS_KEY_ID}"
-        echo "::add-mask::${AWS_SECRET_ACCESS_KEY}"
-        echo "::add-mask::${AWS_SESSION_TOKEN}"
-        echo "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" >> $GITHUB_ENV
-        echo "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" >> $GITHUB_ENV
-        echo "AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}" >> $GITHUB_ENV
-      shell: bash
+    - name: Get temporary AWS credentials
+      uses: armory-io/infra-github-actions/fetch-aws-credentials@master
+      with:
+        vault_addr: '${{ inputs.vault_addr }}'
+        vault_token: '${{ inputs.vault_token }}'
+
```

Comment:
The gradle action like this has `aws_account: hosted-services` but this one doesn't. Is that ok?